RELEASE_NOTES.md

```diff
@@ -41,3 +41,8 @@ also array of versions and array of sources).
 
 Some edge conditions in the `:remove` and `:purge` actions in `dpkg_package` were also fixed and the `:purge` action will now
 purge packages that were previously removed (`apt_package` still does not do this).
+
+
+## `windows_package` now supports non-`MSI` based installers
+
+Today you can install `MSI`s using the `windows_package` resource. However, you have had to use the windows cookbook in order to install non `MSI` based installer packages such as Nullsoft, Inno Setup, Installshield and other `EXE` based installers. We have moved and slightly improved the windows cookbook resource into the core chef client. This means you can now run most windows installer types without taking on external cookbook dependencies.
```

Comment:
we should note in DOC_CHANGES.md that the possible installer_types change.

https://docs.chef.io/resource_windows_package.html